queries.sql

```diff
@@ -72,3 +72,75 @@ between '1990-1-1' and '2000-1-1'; --0
 --escape_attempts
 --------------------
  --2.6666666666666667
+
+update animals set species_id = 2 where name like '%mon';
+update animals set species_id = 1 where name not like '%mon';
+update animals set owner_id = 1 where name = 'Agumon';
+update animals set owner_id = 3 where name = 'Devimon' or  name = 'Plantmon';
+update animals set owner_id = 4 where name = 'Charmander' or  name = 'Squirtle' or name = 'Blossom';
```

Comment:
- [ ] Kindly make sure to add all lines from 76 to 80 in data.sql file.

- [ ] Also enter the queries for :
Modify your inserted animals to include owner information-
- Jennifer Orwell owns Gabumon and Pikachu,
- Dean Winchester owns Angemon and Boarmon.

Reply:
> * [ ]  Kindly make sure to add all lines from 76 to 80 in data.sql file.
> * [ ]  Also enter the queries for :
>   Modify your inserted animals to include owner information-
> * Jennifer Orwell owns Gabumon and Pikachu,
> * Dean Winchester owns Angemon and Boarmon.

@sja-thedude aren't these issues taken care of by the update statements above?
My understanding is that I should update the animals table so Jennifer (for instance) with id:2 will correspond to owner_id:2 in animals and that is reflected by the query below:
`select a.name, o.id,  o.full_name from animals a join owners o on a.owner_id = o.id where o.full_name = 'Jennifer
 Orwell';`
![modify_inserts](https://user-images.githubusercontent.com/5064641/206242677-4b07bcd4-4bc9-4cea-98ab-1b43ea40b8f3.png)